app/retail/templates/shared/activity.html

```diff
@@ -93,7 +130,7 @@
         <img src="{% static row.kudos_data.image %}" alt="" class="w-100">
       </a>
     {% else %}
-      <i class="far {{ row.icon }} last-icon"></i>
+      <i class="fa {{ row.icon }} last-icon"></i>
```

Comment:
why was this change done ? 😅

Reply:
I just like it better like that, you know? ;) It's shorter, simpler, more beautiful.

Reply:
Could you undo this change ? `fa` is retired in `v5` of `font-awesome` which is what we are in now

Reply:
Sure. I'm sorry, I thought you were joking.

Reply:
Haha no worries ^_^